### ISSUE TITLE
Recover passphrase nonfunctional

### ISSUE THREAD

**Issue by b264** [keybase/client#8168]
I forgot my password, unfortunately.
Thankfully, I have a virtual machine I use for development which was still logged-in.
However, after running `keybase passphrase recover` per directions (on the logged-in filesystem) then I still cannot log in with the new password using `keybase login` on the other computer.
The user interface indicates this is what should be done, however it does not work: `Failed to unlock key; bad passphrase`

**Comment by maxtaco** [comment]
Does it work to log into the web site?

**Comment by b264** [comment]
Yes

**Comment by b264** [comment]
Also of note, is that I didn't do anything that should have caused me to have been logged-out... but this is a separate issue I think.

**Comment by maxtaco** [comment]
Can you try to login and then run `keybase log send`?

**Comment by b264** [comment]
`4c01e9f2f30a82a04e2ece1c`

**Comment by b264** [comment]
`▶ NOTICE Stored session expired` was printed when I sent the log.  I have not seen that before

**Comment by maxtaco** [comment]
We'll take a look when we get a chance, sorry about this.

**Comment by patrickxb** [comment]
@b264 Are you still experiencing this?  I looked at the log you sent, but all I see is that the passphrase prompt is canceled:

    2017-08-20T17:40:05.724294-05:00 ▶ [DEBU keybase login_state.go:862] 5e361 | stretchPassphraseIfNecessary: getting keybase passphrase via ui
    2017-08-20T17:40:05.724331-05:00 ▶ [DEBU keybase handler.go:63] 5e362 SecretUI:GetPassphrase, sessionID = 139
    2017-08-20T17:40:08.122705-05:00 ▶ [DEBU keybase login_state.go:849] 5e363 - stretchPassphraseIfNecessary -> ERROR: Input canceled

Also, you can try using your keybase username instead of your email address for login.

**Comment by b264** [comment]
Yes, it says the input was canceled because the window just incessantly pops up asking for my passphrase, immediately after I've entered my passphrase it says it's incorrect and wants me to enter my passphrase in an endless loop.  This works on the web app.  It's still completely broken and unable to be used, the new log is `df4de796aed71250a3638d1c`

**Comment by b264** [comment]
At this point, I am migrating stuff off keybase because if it stops working on my one virtual machine I will lose everything.

**Comment by b264** [comment]
If you need any information to help find this issue, please ask

**Comment by patrickxb** [comment]
Does kbchat work for you in your vm?  Can we chat there and try to resolve this?  I'm `patrick`.  I'll try sending you a message.

**Comment by b264** [comment]
```
devbox@ubuntu-server-xenial:~$ kbchat
kbchat: command not found
devbox@ubuntu-server-xenial:~$ 
```

**Comment by b264** [comment]
`4db688e5a6179a486aed191c`
